volatility/framework/symbols/windows/extensions/__init__.py

```diff
@@ -666,3 +700,81 @@ def to_list(self,
 
     def __iter__(self) -> Iterator[interfaces.objects.ObjectInterface]:
         return self.to_list(self.vol.parent.vol.type_name, self.vol.member_name)
+
+
+class TOKEN(objects.StructType):
+    """A class for process etoken object."""
+
+    def get_sids(self) -> Iterable[str]:
+        """Yield a sid for the current token object."""
+
+        userAndGroupCount = int(self.UserAndGroupCount)
+        if userAndGroupCount < 0xFFFF:
+            layer_name = self.vol.layer_name
+            kvo = self._context.layers[layer_name].config["kernel_virtual_offset"]        
+            symbol_table = self.get_symbol_table_name()
+            ntkrnlmp = self._context.module(symbol_table,
+                                      layer_name = layer_name,
+                                      offset = kvo)
+            UserAndGroups = ntkrnlmp.object(object_type="array",
+                                       offset=self.UserAndGroups.dereference().vol.get("offset") - kvo,
```

Comment:
`vol.get` is a bit of a waste, there's no default (meaning if it doesn't exist, you'll get back a `None` which won't subtract, and it cleaner just to write `.vol.offset`.